zerver/openapi/zulip.yaml

```diff
@@ -11498,6 +11504,14 @@ paths:
 
                           **Changes**: New in Zulip 3.0 (feature level 11). Previously this
                           value was hardcoded to 86400 seconds (1 day).
+                      realm_enable_read_receipts:
+                        type: boolean
+                        description: |
+                          Present if `realm` is present in `fetch_event_types`.
+
+                          Whether read receipts is enabled in the organization or not.
```

Comment:
We should add another sentence: "If disabled, read receipt data will be unavailable to clients, regardless of individual users' personal read receipt settings."